test/fast/environment-test.ts

```diff
@@ -17,4 +18,21 @@ describe('environment variables', () => {
     })
     expect(result.stdout).to.equal('Foo Bar <foo@bar.com> 1475703207 +0200\n')
   })
+
+  it('when GIT_EXEC_PATH environment variable is *not* set, it will be calculated', async () => {
+    expect(process.env.GIT_EXEC_PATH).to.be.undefined
```

Comment:
While this is a neat guard clause to confirm we're in the right state, I think this reflects some poor design choices by me. I'd like to get to the point where the tests don't explicitly rely on `process.env` and `process.platform`, and you can control the test by passing in different values, but I won't hold up this PR.